ui/components/GrafanaCustomChart.js

```diff
@@ -754,7 +758,7 @@ class GrafanaCustomChart extends Component {
       } else {
         mainChart = (
           <div>
-            <div className={classes.error}>{error && 'There was an error communicating with the server'}</div>
```

Comment:
Any reason to remove the error explicitly?

Reply:
I missed it its was for testing! 
Thanks